packages/docs/content/foundation/system-color-tokens.mdx

```diff
@@ -1,7 +1,9 @@
 ---
 title: System color tokens
 order: 40
-intro: System color tokens are the complete set of color tokens from which themes are built.
+intro: System color tokens are the entire set of color tokens from which our themes are built.
 ---
 
+However, when building your product, it's essential to use the theme color tokens that have been defined for your Brand's experience. Think of System color tokens like an entire box of crayons, our themes use specific colors from the box, not all of them.
```

Comment:
Because the text above exists in a `<header>` and the text below exists in an `<article>` that is outside of that `<header>`, I don't know if it's a guarantee that our users will be reading both sections together. I may be wrong. In light of that, though, maybe some additional context at the start of this paragraph might be more appropriate than starting with _However_. Like what if it started with "While this page shows the full range of colors, when building your product...

Reply:
Ah, gotcha. And now that I'm looking at the code that's shedding more light on our spacing conversation too— thank you.